applications/MeshingApplication/tests/cpp_tests/test_internal_interpolation_process.cpp

```diff
@@ -623,8 +623,8 @@ namespace Kratos
 
             Properties::Pointer p_elem_prop = r_model_part.CreateNewProperties(0);
             // In case the StructuralMechanicsApplciation is not compiled we skip the test
-            if (!KratosComponents<ConstitutiveLaw>::Has("LinearElastic3DLaw"))
-                return void();
+            KRATOS_SKIP_TEST_IF_NOT(KratosComponents<ConstitutiveLaw>::Has("LinearElastic3DLaw")) << "This test needs the StructuralMechanicsApplication" << std::endl;
```

Comment:
Since we are thinking of moving this to the core I would use sth else
Probably the solid-elements are suited best

Reply:
At the end is the same, the important thing is to check is the law/element exists. See that I am doing the same check than before

Reply:
Done, now is with element